pkg/cvo/sync_worker.go

```diff
@@ -435,11 +435,13 @@ func (w *SyncWorker) Update(ctx context.Context, generation int64, desired confi
 				Image:   work.Desired.Image,
 			},
 		}
+		w.work = work
```

Comment:
setting this before the `w.loadUpdatedPayload` feels weird.  Why do we need this?

Reply:
It is weird. Wanted to get `Capabilities` saved since they can change independent of payload. Changed such that I only save capabilities and not all of work.